### DISCUSSION TITLE
Clashes when switching from full screen to windowed after switching to different screen

### DISCUSSION THREAD

**Discussion by Xeladoes** [ppy/osu#22867] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

I'm playing the newest version of osu!lazer on windows10;

Here is how to trigger the problem:
1. Turn on Instant Replay of Nvidia Geforce Experience
2. Switch osu!lazer from one screen to another, then maybe switch back, using mouse or hotkey: win+ArrowKey(you  need to have more 2 screen to trigger the bug)
3. Switch to fullscreen or windowed or borderless, then game will stucked and crashed after a while.

Yes if you turn Instant Replay of Nvidia Geforce Experience off, everything will be fine.

### Version

osu! 2023.207.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/10955837/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10955838/updater.log)
[database.log](https://github.com/ppy/osu/files/10955839/database.log)
[input.log](https://github.com/ppy/osu/files/10955840/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10955841/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10955842/network.log)
[performance.log](https://github.com/ppy/osu/files/10955843/performance.log)

**Comment by Xeladoes** [comment]
I'm not so sure on setp 1 and 2, anyway if you switch screen and switch window mode over and over again, it will stuck after a few turns.

**Comment by peppy** [comment, marked as answer]
Will probably be resolved once we change the default renderer to D3D.
